src/test/java/com/google/api/generator/gapic/protoparser/BUILD.bazel

```diff
@@ -45,6 +46,7 @@ filegroup(
         "//src/test/java/com/google/api/generator/gapic/testdata:showcase_java_proto",
         "//src/test/java/com/google/api/generator/gapic/testdata:testgapic_java_proto",
         "@com_google_api_api_common//jar",
+        "@com_google_api_grpc_proto_google_common_protos",
```

Comment:
You should not be needing this, as this dependency must be pulled transitively (many stuff in protoparser package already depends on this artifact). What fails if you remove it?

Reply:
I just tried and nothing broke, I'll remove it.